notebooks/photutils/03_aperture_photometry/03_aperture_photometry.ipynb

```diff
@@ -0,0 +1,970 @@
+    "\n",
+    "And, better yet, our `CCDData` object has held onto its WCS information ever since we created it!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wcs = xdf_image.wcs\n",
+    "sky_elliptical_apertures = [ap.to_sky(wcs) for ap in elliptical_apertures]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we wanted to generate `SkyEllipticalAperture` objects from the get-go, we could have used that WCS object in the following way:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from photutils import SkyEllipticalAperture\n",
+    "from astropy.coordinates import SkyCoord\n",
+    "\n",
+    "r = 3.  # pixels; approximate isophotal extent of semimajor axis\n",
+    "\n",
+    "# Create the apertures\n",
+    "sky_elliptical_apertures = []\n",
+    "for obj in catalog:\n",
+    "    # Convert the centroids into RA/Dec using WCS\n",
+    "    ra, dec = wcs.all_pix2world(obj.xcentroid.value, obj.ycentroid.value, 0)\n",
+    "    # Convert the positions to an Astropy SkyCoord object, with units!\n",
+    "    sky_position = SkyCoord(ra, dec, unit=u.deg)\n",
+    "    \n",
+    "    # Define the elliptical parameters, now with units\n",
+    "    a = obj.semimajor_axis_sigma.value * r * u.pix\n",
+    "    b = obj.semiminor_axis_sigma.value * r * u.pix\n",
+    "    theta = obj.orientation.value  * u.rad\n",
+    "    \n",
+    "    # Convert the theta from radians from X axis to the radians from North \n",
+    "    x_to_north_angle = (90. + header['ORIENTAT']) * u.deg\n",
+    "    x_to_north_angle_rad = x_to_north_angle.to_value(u.rad) * u.rad\n",
+    "    theta -= x_to_north_angle_rad\n",
+    "    \n",
+    "    # Define the apertures\n",
+    "    ap = SkyEllipticalAperture(sky_position, a, b, theta=theta)\n",
+    "    sky_elliptical_apertures.append(ap)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Unfortunately, you can't plot SkyApertures. However, you can use them just as easily to perform aperture photometry!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----\n",
+    "## Performing Aperture Photometry"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have aperture objects that fit our data reasonably well, we can finally perform photometry with the `aperture_photometry` function. This function takes the following arguments:\n",
+    "\n",
+    "* **`data`** - the background-subtracted data array on which to perform photometry.\n",
```

Comment:
If you changed the other time I mentioned changing these hyphens to n-dashes these should change as well; otherwise if you ignored me the first time feel free to ignore me the second time!